pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -271,7 +255,7 @@ func (r *ReconcileWindowsMachineConfig) removeWorkerNode(vm types.WindowsVM) Rec
 	// VM is missing credentials, this can occur if there was a failure initially creating it. We can consider the
 	// actual VM terminated as there is nothing we can do with it.
 	if vm.GetCredentials() == nil || len(vm.GetCredentials().GetInstanceId()) == 0 {
-		delete(r.windowsVMs, vm)
+		delete(r.tracker.windowsVMs, vm)
```

Comment:
Now that `windowsVMs` is part of the `tracker` struct, I am wondering deleting or adding to it should also be done within `tracker`.

Reply:
Im in favor of that change (add and delete functions)

Reply:
Ok, I'll add it.

Reply:
I don't see this change. Did you forget to push?